Pittsburgh-Light-Rail-App/LaunchSystem.py

```diff
@@ -65,6 +66,60 @@ def __init__(self, hw):
         file = f'{os.getcwd()}/train.ui'
         self.trainModel = TrainModel(file, hw, self.signals)
         self.globalOcc = []
+        self.signals.timeSignal.connect(self.showTime)
```

Comment:
For some reason, it won't let me comment on lines that haven't been changed, but you should add your name to the top of this file as an author.  Everyone should do that as some point since we've all worked on this.